themes/base/tooltip.css

```diff
@@ -13,8 +13,8 @@
 	position: absolute;
 	z-index: 9999;
 	max-width: 300px;
-	-webkit-box-shadow: 0 0 5px #aaa;
-	box-shadow: 0 0 5px #aaa;
+	-webkit-box-shadow: 0 0 5px #ddd;
+	box-shadow: 0 0 5px #ddd;
```

Comment:
Ah, right. Nevermind. I was just looking over this and saw the last comment saying you were going to make a change last week.